### ISSUE TITLE
@OneToMany mapping with new syntax not working

### ISSUE THREAD

**Issue by lookfirst** [mikro-orm/mikro-orm#158]
![image](https://user-images.githubusercontent.com/85355/65448677-9c7bd300-de63-11e9-8a23-5c0276c53d49.png)

In my example project, I'm not sure how to map things using the new syntax. It seems like this should work.

**Comment by B4nan** [comment]
Maybe you have some problem in the other entity definition? Will take a closer look tomorrow.

**Comment by lookfirst** [comment]
Entirely possible.

```
Thing:
        what you see above.

Dog:
	@ManyToOne()
	thing!: Thing;
```

**Comment by B4nan** [comment]
Lol, I forgot to change the typings on others operators, it's only on `@OneToOne` decorators now :D It should work, this is just typings issues so you can bypass it via `any` in the meantime (or put string there instead of callback).

**Comment by lookfirst** [comment]
Yea, that is kind of what I suspected, so I tested it and filed the issue. ;-)

**Comment by B4nan** [comment]
Fixed in `3.0.0-alpha.14`.
